### PR TITLE
Fix wrong translations and add pending.

### PR DESCRIPTION
**Pull request by ericfab179**

### CODE REVIEW

File: lms/static/js/i18n/fr/djangojs.js [hunk 1]
```diff
@@ -659,6 +659,7 @@
     "Endorse": "Approuver", 
     "Ends {end}": "Finit \u00e0 {end}", 
     "Engage with posts": "Participez \u00e0 la conversation", 
+    "Enrol Now": "Inscrivez-vous maintenant",
```
Comment (jfavellar90) [inline lms/static/js/i18n/fr/djangojs.js:662]: the file cms/static/js/i18n/fr/djangojs.js is missing

File: conf/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -27852,3 +27848,13 @@ msgstr "Emplacement du problème: "
...
 msgstr "Télécharger le CSV des étudiants Statut des certificats"
+
+#: Found in edx-theme-microsites and localized
+#: GCS/templates/footer.html
+msgid "Terms of Service and Honour Code"
+msgstr "Conditions d'utilisation et Code d'honneur"
+
+#: Found in edx-theme-microsites and localized
+#: GCS/templates/courseware/course_about.html
+msgid "Enrol Now"
```
Comment (jfavellar90) [inline conf/locale/fr/LC_MESSAGES/django.po:27859]: why the typo on "Enrol" (double l)? is it because they have the same typo on the microsites repo?

Comment (ericfab179) [reply conf/locale/fr/LC_MESSAGES/django.po:27859]: This is not a typo, this is the source of our problems jaja https://writingexplained.org/enrolment-vs-enrollment-difference British vs American

File: lms/static/js/i18n/fr/djangojs.js (hunk 1 above)
Comment (ericfab179) [reply lms/static/js/i18n/fr/djangojs.js:662]: Done

File: conf/locale/fr/LC_MESSAGES/django.po (hunk 2 above)
Comment (jfavellar90) [reply conf/locale/fr/LC_MESSAGES/django.po:27859]: A complete document about this, wauuuuu

**Review by jfavellar90 — APPROVED** [review]

**Pull request merged**